src/__test__/redux/reducers/samplesReducer.test.js

```diff
@@ -0,0 +1,90 @@
+import samplesReducer from '../../../redux/reducers/samples';
+import initialState, { sampleTemplate, sampleFileTemplate } from '../../../redux/reducers/samples/initialState';
+
+import {
+  SAMPLES_CREATE, SAMPLES_UPDATE,
+} from '../../../redux/actionTypes/samples';
+
+describe('samplesReducer', () => {
+  const sample1Files = {
+    fileNames: ['features.tsv'],
+    files: {
+      'features.tsv': {
+        ...sampleFileTemplate,
+        name: 'features.tsv',
+      },
```

Comment:
I am not sure if adding the files to a test sample is of any use. The same applies to creating the `sampleFileTemplate` in the reducer's initial state.